packages/client/src/checkout/draftOrders/types/postDraftOrders.types.ts

```diff
@@ -0,0 +1,15 @@
+import type { Config } from '../../../types/index.js';
+import type { DraftOrder } from './draftOrder.types.js';
+
+export type PostDraftOrdersData = {
+  orderId: number;
+  customerId?: string;
+  paymentMethods?: string[];
+};
+
+export type PostDraftOrdersResponse = DraftOrder;
```

Comment:
I think we can remove this alias and have only `DraftOrder` on the postDraftOrders.types.ts.

Reply:
Removed 👍